langlib/lang.stream/src/main/ballerina/stream.bal

```diff
@@ -91,6 +139,25 @@ public isolated function 'map(stream<Type,CompletionType> stm, @isolatedParam fu
 # The combining function takes the combined value so far and a member of the stream,
 # and returns a new combined value.
 #
+# ```ballerina
+# Employee[] employeeList = [
+#     {employeeID: "AA1298", salaryInUsd: 5000},
+#     {employeeID: "BB4562", salaryInUsd: 3000},
+#     {employeeID: "CC2398", salaryInUsd: 4500}
+# ];
+#
+# stream<Employee> employeeStream = employeeList.toStream();
+# decimal expenseWithoutPayroll = 20000;
```

Comment:
By looking at the above examples, we need a new line after this to be consistent with other examples, wdyt?